File: src/intellibitz/sted/actions/OpenFontMapAction.java
package intellibitz.sted.actions;

import intellibitz.sted.ui.STEDWindow;

import java.awt.event.ActionEvent;

public class OpenFontMapAction
        extends STEDWindowAction {
    public OpenFontMapAction() {
        super();
    }

    public void actionPerformed(ActionEvent e) {
        final STEDWindow stedWindow = getSTEDWindow();
//        STEDGUI.busy();
//        if (JOptionPane.CANCEL_OPTION !=
//                stedWindow.getDesktop().saveDirty())
//        {
        stedWindow.getDesktop().openFontMap();
//        }
//        STEDGUI.relax();
    }

}

File: src/main/intellibitz/sted/actions/OpenFontMapAction.kt
package sted.actions

import java.awt.event.ActionEvent

open class OpenFontMapAction : STEDWindowAction() {
    override fun actionPerformed(e: ActionEvent) {
        //        STEDGUI.busy();
//        if (JOptionPane.CANCEL_OPTION !=
//                stedWindow.getDesktop().saveDirty())
//        {
        stedWindow.desktop.openFontMap()
        //        }
//        STEDGUI.relax();
    }
}